app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,16 @@
+class LikesController < ApplicationController
+  def new
+    @like = Like.new
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @like = @post.likes.new(author_id: current_user.id, post_id: @post.id)
+
+    if @like.save
+      redirect_to root_path
```

Comment:
- [ ] Please redirect to the post path instead of the root path

Example: ` redirect_to @post`

Reply:
This issue is well resolved! ✔️